### PR TITLE
World Metric App for PR

### PR DESCRIPTION
**Pull request by supreeti**
In this Web Application  :-
- Used React documentation.
- Used React components.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handled events in a React app.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Used React life cycle methods.
- Applied React best practices and language style guides in code.
- Used store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,170 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to Original design idea by [https://www.behance.net/sakwadesignstudio](Nelson Sakwa on Behance.)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/supreeti/Metric-webapp/blob/main/LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rotimiazeez) [inline README.md:184]: Please ensure you record a (not more than) 5 min video presenting the features of this project, and add the presentation link in your README file 👍🏽

Comment (rotimiazeez) [reply README.md:184]: Please ensure your app is deployed and available online, and add the deployed link to your README file 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @supreeti  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good code structure 👍🏽
- App functioning as required 💯 
- Good job giveing appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). ✌🏽
- Good app UI 👏🏽
- Tests are implemented and passing 💪🏽
- No Linter error ✔️ 
- Good PR title & description 👍🏽
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @supreeti    , 

Great Job implementing all the Previous reviewer's suggestions :clap: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes Implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**